dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/liberty/CDIRuntimeImpl.java

```diff
@@ -487,6 +493,13 @@ public void applicationStopped(ApplicationInfo appInfo) {
             } catch (CDIException e) {
                 //FFDC and carry on
             }
+            // Clean up the application TCCL created for startup
```

Comment:
This block should be in a finally block to avoid possible memory leak

Reply:
Done